libbeat/processors/dissect/field.go

```diff
@@ -161,7 +242,7 @@ type indirectField struct {
 func (f indirectField) Apply(b string, m Map) {
 	v, ok := m[f.Key()]
 	if ok {
-		m[v] = b
+		m[v.(string)] = b
```

Comment:
For safety, to avoid panics, I think we should do some type check here. Maybe an option is to do nothing if the value is not a string.
```suggestion
		if v, ok := v.(string); ok {
			m[v] = b
		}
```

Reply:
Updated to use code above.